kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -67,6 +68,7 @@ def check(self, _):
         self.report_consumer_offsets_and_lag(
             consumer_offsets, highwater_offsets, self._context_limit - len(highwater_offsets)
         )
+        self.client.close_admin_client()
```

Comment:
Unlike the changelog entry, this doesn't appear to be configurable. Can we add an option?

Reply:
Added a config option for it.